docs/content/doc/installation/with-docker.en-us.md

```diff
@@ -253,7 +253,8 @@ You can configure some of Gitea's settings via environment variables:
 
 * `APP_NAME`: **"Gitea: Git with a cup of tea"**: Application name, used in the page title.
 * `RUN_MODE`: **dev**: For performance and other purposes, change this to `prod` when deployed to a production environment.
-* `SSH_DOMAIN`: **localhost**: Domain name of this server, used for the displayed clone URL in Gitea's UI.
+* `DOMAIN`: **localhost**: Domain name of this server, used for the displayed clone URL in Gitea's UI.
+* `SSH_DOMAIN`: **""**: Domain name of this server, used for the displayed clone URL in Gitea's UI.
```

Comment:
```suggestion
* `DOMAIN`: **localhost**: Domain name of this server, used for the displayed http clone URL in Gitea's UI.
* `SSH_DOMAIN`: **""**: Domain name of this server, used for the displayed ssh clone URL in Gitea's UI.
```
Just to make it clear what the difference is.

Reply:
but during the /install page, the line for the SSH Domain Server takes the DOMAIN variable

Reply:
I aggree with @gary-kim

Reply:
> but during the /install page, the line for the SSH Domain Server takes the DOMAIN variable

Not necessarily exactly this. Just want to make sure no one asks why there are two different environment variables for the same setting.

Reply:
ok, thanks for the feedback. I have done some modifications.